mifosng-android/src/main/res/layout/fragment_add_savings_account.xml

```diff
@@ -87,7 +88,9 @@
                 android:imeOptions="actionNext"
                 android:paddingBottom="16dp"
                 android:paddingTop="10dp"
-                android:textSize="20sp" />
+                android:textSize="14sp"
```

Comment:
It would be great if you could remove the hardcoded dimensions in this file and put them ```dimens.xml```

Reply:
Done